build/linux/Dockerfile

```diff
@@ -5,4 +5,10 @@ WORKDIR /app
 COPY dist /app/
 COPY static /app/static
 
+# TODO: REVIEW
+# Health check only available on Linux containers for the scope of the POC
```

Comment:
the only change needed to make it available in other types is copy-pasting L12, right?

Reply:
I think that we'll need to change the CMD but I never defined a healthcheck on a Windows container